### ISSUE TITLE
Some fixes for options in constructor

### ISSUE THREAD

**Issue by frct1** [schroffl/teamspeak-query#7]
Hi :)
Today i was need connect to SQ interface from custom ip and i'm seeing lib source. And i have the one fix for this.
Assign ip to connect we can only in connect method, but in library there pass only host and port => we cann pass custom ip to connect.
passing options to `new net.Socket(options) didn't do enough actions.
this is fixes from node js noob, but it pretty works

```
constructor(host, port, options={}) {
[some code]
Object.assign(options,{port:port,host:host});
sock.connect(options);
```
Then we can connect to sq with from custom ip by this code
`new TeamspeakQuery(server.ip, server.query,{localAddress:'my.ip.123.322'});`
I'm sure that you can offer a more intelligent version of this function :smile:

**Comment by schroffl** [comment]
You want to be able to pass options to the `net.Socket.connect` function?

**Comment by frct1** [comment]
Yes, for greater flexibility in connection parameters to SQ.
I think it will useful for future users this lib

**Comment by schroffl** [comment]
Published in [v2.0.0](https://github.com/schroffl/teamspeak-query/releases/tag/v2.0.0).

**Comment by frct1** [comment]
You are the best. Thanks for the quick response.
